experiments/20160805-mutagenesis/20160902-mutagenesis-pb2.md

```diff
@@ -0,0 +1,38 @@
+## Goals
+
+- generate mutant PB2 PCR products with full DNA fragment
```

Comment:
May want to change this before I make the merge. It's important to make sure that the experiment goals are documented clearly.